arch/arm/boot/dts/imx6ul-ts7250v3.dts

```diff
@@ -0,0 +1,1200 @@
+		"MAGNET_IRQ",
+		"",
+		"CONSOLE_TXD",
+		"CONSOLE_RXD",
+		"EN_RED_LED#",
+		"EN_GRN_LED#",
+		"",
+		"EN_XBEE_USB",
+		"CAN_2_TXD",
+		"CAN2_RXD_3V",
+		"UART3_TXD",
+		"UART3_RXD",
+		"UART3_CTS#",
+		"UART3_RTS#",
+		"UART4_TXD",
+		"UART4_RXD",
+		"EN_DIO_FET",
+		"NIM_PWR_ON";
+};
+
+&gpio2 {
+	gpio-line-names =
+		"ENET1_RX_DATA0",
+		"ENET1_RX_DATA1",
+		"ENET1_RX_EN",
+		"ENET1_TX_DATA0",
+		"ENET1_TX_DATA1",
+		"ENET1_TX_EN",
+		"ENET1_TX_CLK",
+		"ENET1_RX_ER",
+		"ENET2_RX_DATA0",
+		"ENET2_RX_DATA1",
+		"ENET2_RX_EN",
+		"ENET2_TX_DATA0",
+		"ENET2_TX_DATA1",
+		"ENET2_TX_EN",
+		"ENET2_TX_CLK",
+		"ENET2_RX_ER",
+		"EMMC_CMD",
+		"EMMC_CLK",
+		"EMMC_D0",
+		"EMMC_D1",
+		"EMMC_D2",
+		"EMMC_D3",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"";
+};
+
+&gpio3 {
+	gpio-line-names =
+		"EN_USB_5V",
+		"WIFI_IRQ",
+		"",
+		"UART4_CTS#",
+		"WDOG#",
+		"I2C_3_DAT",
+		"I2C_3_CLK",
+		"ISA_RESET",
+		"ISA_IOCHK",
+		"LCD_PIN7",
+		"LCD_PIN8",
+		"LCD_PIN9",
+		"LCD_PIN10",
+		"CAN_1_TXD",
+		"CAN1_RXD_3V",
+		"LCD_PIN11",
+		"LCD_PIN12",
+		"LCD_PIN13",
+		"LCD_PIN14",
+		"LCD_WR#",
+		"LCD_EN",
+		"LCD_RS",
+		"SYS_RESET#",
+		"SD_CMD",
+		"SD_CLK",
+		"SD_D0",
+		"SD_D1",
+		"SD_D2",
+		"SD_D3",
+		"",
+		"",
+		"";
+};
+
+&gpio4 {
+	gpio-line-names =
+		"FPGA_FLASH_SELECT",
+		"DETECT_94-120",
+		"EIM_AD_08",
+		"EIM_AD_09",
+		"EIM_AD_10",
+		"EIM_AD_11",
+		"EIM_AD_12",
+		"EIM_AD_13",
+		"EIM_AD_14",
+		"EIM_AD_15",
+		"EIM_IRQ",
+		"EIM_BLK",
+		"SPI_3_CS#",
+		"SPI_3_CLK",
+		"SPI_3_MOSI",
+		"SPI_3_MISO",
+		"EIM_WAIT#",
+		"EIM_CS0#",
+		"EIM_OE#",
+		"EIM_WE#",
+		"EIM_LBA#",
+		"EIM_AD_00",
+		"EIM_AD_01",
+		"EIM_AD_02",
+		"EIM_AD_03",
+		"EIM_AD_04",
+		"EIM_AD_05",
+		"EIM_AD_06",
+		"EIM_AD_07",
+		"",
+		"",
+		"";
+};
+
+&gpio5 {
+	gpio-line-names =
+		"GYRO_INT",
+		"FPGA_IRQ",
+		"EN_EMMC_3.3V",
+		"GPIO_DVFS",
+		"",
+		"WIFI_RESET#",
+		"EN_WIFI_PWR",
+		"EN_CL_1",
+		"EN_CL_2",
+		"EN_CL_3",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"",
+		"";
+};
+
+&i2c3 {
+	clock-frequency = <100000>;
+	pinctrl-names = "default", "gpio";
+	pinctrl-0 = <&pinctrl_i2c3>;
+	pinctrl-1 = <&pinctrl_i2c3_gpio>;
+	scl-gpios = <&gpio3 06 GPIO_ACTIVE_HIGH>;
+	sda-gpios = <&gpio3 05 GPIO_ACTIVE_HIGH>;
+	status = "okay";
+
+	pc104gpio: pca9555@20 {
+		compatible = "nxp,pca9555";
+		reg = <0x20>;
+		gpio-controller;
+		#gpio-cells = <2>;
+
+		gpio-line-names =
+			"ISA_CN_D03",
+			"ISA_CN_D04",
+			"ISA_CN_D05",
+			"ISA_CN_D06",
+			"ISA_CN_D07",
+			"ISA_CN_D08",
+			"ISA_CN_D09",
+			"ISA_CN_D10",
+			"ISA_CN_D11",
+			"ISA_CN_D12",
+			"ISA_CN_D13",
+			"ISA_CN_D14",
+			"ISA_CN_D15",
+			"",
+			"",
+			"";
+	};
+};
+
+&uart1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart1>;
+	status = "okay";
+};
+
+&uart3 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart3>;
+	fsl,uart-has-rtscts;
```

Comment:
This is just `uart-has-rtscts` the fsl specific property is no longer used. Same issue below in uart4